doc-content/enterprise-only/admin-and-config/managing-business-central-artifacts-proc.adoc

```diff
@@ -0,0 +1,36 @@
+[id='managing-business-central-artifacts-proc']
+= Managing artifacts
+
+You can manage artifacts from the *Artifacts* page in {CENTRAL}. The artifact repository is a Maven repository and there is only one Maven repository for each installation.
```

Comment:
Important to mention that this UI is for the *local* maven repository that BC has, and ideally this shouldn't be used, better always use a proper maven repository solution like nexus, artifactory etc.

Reply:
Changed as suggested.

Reply:
Changed as suggested.